proj_utils/Parser.py

```diff
@@ -0,0 +1,75 @@
+from time import sleep
+
+from models.Payload import Payload
+import os
+import json
+
+
+class Parser:
+    def __init__(self, payload: Payload):
+        self.payload = payload
+        self.jsonObject = {}
+
+    def create_text_file(self):
+        save_path = 'C:/Users/razm1/PycharmProjects/routeExercise/students'
+        fullname = f"{self.jsonObject['studentDetails']['fullName']}"
+        complete_path = os.path.join(save_path, fullname.replace(' ', '_') + ".txt")
+
+        with open(complete_path, 'w') as f:
+            data = json.dumps(self.jsonObject, indent=4)
+            f.write(data)
+
+    def populate_json_object(self):
+        self.populate_student_details()
+        self.populate_subject_grades()
+        self.populate_total_avg()
+        self.populate_birth_date()
+        self.populate_age()
+        self.populate_gender()
+        self.populate_isgoodbehavior()
+        self.populate_notes()
+
+    def populate_student_details(self):
+        self.jsonObject["studentDetails"] = {}
+        self.jsonObject["studentDetails"].update({
+            "firstName": f"{self.payload.studentDetails['firstName']}",
```

Comment:
you use hard coded values for field names a lot. put them in a configuration

Reply:
should I use XML file for configuration? or a python class? which one is considered best approach ?